### ISSUE TITLE
Identifier: Update Accessibility link and guidance

### ISSUE THREAD

**Issue by thisisdano** [uswds/uswds#5092]
## Description

**Improve accuracy of Accessibility link.** Include a more accurate accessibility statement link in the identifier and update the guidance to reflect the details of the requirement.

Our Identifier should support the OMB memorandum ["Strategic Plan for Improving Management of Section 508 of the Rehabilitation Act"](https://obamawhitehouse.archives.gov/sites/default/files/omb/procurement/memo/strategic-plan-508-compliance.pdf) [PDF, 212 KB] by linking to their agency's accessibility statement. Section508.gov has [more information about the accessibility statement](https://www.section508.gov/manage/laws-and-policies/website-accessibility-statement/). This accessibility statement includes Section 508 program and contact information, but should not be considered "accessibility support" in the conventional user-facing understanding.

Our guidance (and [the required links guidance on Digital.gov](https://digital.gov/resources/required-web-content-and-links/#accessibility-statement)) indicates that this link should be called `Accessibility` but the Identifier currently uses `Accessibility support`. 

Since this link is a link to an accessibility statement, our Identifier should use `Accessibility statement`.

And finally, our guidance says that this should "Link to the “Accessibility” page of your parent agency’s principal website" when what it should say is "Link to your agency's [accessibility statement](https://www.section508.gov/manage/laws-and-policies/website-accessibility-statement/)." Every agency should have an accessibility statement, any teams should not necessarily link to a parent agency, simply their own agency. 

### What we need to do

- [x] Update the Identifier link should read `Accessibility statement` instead of `Accessibility support`.
- [ ] Contact Digital.gov and suggest that their guidance should use `Accessibility statement` instead of `Accessibility`.
- [x] Update each version of the Identifier
- [x] Update the Spanish versions of the Identifier
- [x] Update [our Identifier guidance](https://designsystem.digital.gov/components/identifier/) to read "Link to your agency's [accessibility statement](https://www.section508.gov/manage/laws-and-policies/website-accessibility-statement/)." in the Accessibility section
- [x] Update the example link in the guidance to a more conventional example of an accessibility statement, [the accessibility statement from DHS](https://www.dhs.gov/accessibility).
- [x] Update changelog for the Identifier and note this change at a monthly call and in the newsletter

## Steps to reproduce the issue

1. Look at our [Identifier page](https://designsystem.digital.gov/components/identifier/) and confirm that the Accessibility link reads `Accessibility support`
2. Look at the current Accessibility Statement guidance from Section508.gov
